aten/src/ATen/native/native_functions.yaml

```diff
@@ -1127,3 +1127,48 @@
   dispatch:
     CPU: _s_poisson_cpu
     CUDA: _s_poisson_cuda
+
+- func: b_add(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_sub(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_mul(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_add(TensorList self, Scalar other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_sub(TensorList self, Scalar other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_mul(TensorList self, Scalar other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_tanh(TensorList self) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_relu(TensorList self) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_sigmoid(TensorList self) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_mm(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_matmul(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_contiguous(TensorList self) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_view(TensorList self, IntList sizes_new) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_update(TensorList self, TensorList other) -> (Tensor, Tensor, Tensor)
+  variants: function
+
+- func: b_synchronize(TensorList self) -> (Tensor, Tensor, Tensor)
```

Comment:
Currently those will get exposed in the `torch.` namespace, but that's surely something we do not want to do. If we really want to keep them in C++, add a leading underscore.

Reply:
How can we call these (for e.g. in tests) if they're hidden?

Reply:
`torch._C._b_synchronize` or `from torch._C import _b_synchronize`

Reply:
What I imagined we would somehow do (I don't know if it's a good idea, or even possible) is to create a new namespace that's accessible to Python as methods on a `Batch` type, and also overload `torch.` functions when called on objects that happen to be batches.